questions.js

```diff
@@ -1,74 +1,46 @@
-'use strict';
-var questions = [],
-    fs = require('fs'),
-	JSON5 = require('json5'),
-    correctNameEnding,
-    jsonData;
 
-var originCountries = ['Україна', 'Англія', 'Франція', 'Іспанія', 'Німеччина', 'Чехія', 'Молдова', 'Уельс', 'Китай', 'Азербайджан', 'Чилі'];
```

Comment:
why did you remove this code? We still need to do this.

You can create your test questions in the different file.